tools/topology/topology2/include/components/micsel.conf

```diff
@@ -0,0 +1,101 @@
+#
+# Common widget micsel
```

Comment:
"It Combines N number of channels at
input into M number of channels at output." Is this really correct? It is only a selector isnt it? There's no combining here